web-app/package.json

```diff
@@ -4,14 +4,15 @@
   "version": "0.0.1",
   "scripts": {
     "tailwind": "npx tailwind build static/tailwind.config.css -o static/tailwind.css",
+    "protoc": "protoc -I=../proto pubgolf.proto --js_out=import_style=commonjs:./src/proto --grpc-web_out=import_style=commonjs,mode=grpcwebtext:./src/proto",
```

Comment:
I think this might need to be updated with the new command I added in #6, which builds the whole proto directory. Sorry about that.